### PR TITLE
Set --prefix only for format-coverage subcommand.

### LINKED ISSUE #117: Coverage.py (python)
I am trying to get codeclimate-action to work with coverage.py. However I am having some issues. I switched from conda to pip which solved the issue that the command coverage isn't found. However now I have some issues with the action mangling paths it seems:

```
/home/runner/work/peaksql/peaksql/cc-reporter before-build
/opt/hostedtoolcache/Python/3.7.6/x64/bin/coverage xml
/home/runner/work/peaksql/peaksql/cc-reporter after-build --exit-code 0 --debug --prefix 
time="2020-02-26T08:17:55Z" level=debug msg="about to run format-coverage" 
time="2020-02-26T08:17:55Z" level=debug msg="searching for a formatter to use" 
time="2020-02-26T08:17:55Z" level=debug msg="checking cobertura formatter" 
time="2020-02-26T08:17:55Z" level=debug msg="checking search path cobertura.xml for cobertura formatter" 
time="2020-02-26T08:17:55Z" level=debug msg="checking coverage.py formatter" 
time="2020-02-26T08:17:55Z" level=debug msg="checking search path coverage.xml for coverage.py formatter" 
time="2020-02-26T08:17:55Z" level=debug msg="found file coverage.xml for coverage.py formatter" 
time="2020-02-26T08:17:56Z" level=debug msg="couldn't load committed at from ENV, trying git..." 
time="2020-02-26T08:17:56Z" level=debug msg="creating test file report for /home/runner/work/peaksql/peaksql/peaksql/__init__.py" 
time="2020-02-26T08:17:56Z" level=info msg="trimming with prefix /" 
time="2020-02-26T08:17:56Z" level=debug msg="getting fallback blob_id for source file home/runner/work/peaksql/peaksql/peaksql/__init__.py" 
time="2020-02-26T08:17:56Z" level=error msg="failed to read file home/runner/work/peaksql/peaksql/peaksql/__init__.py\nopen home/runner/work/peaksql/peaksql/peaksql/__init__.py: no such file or directory" 
Error: open home/runner/work/peaksql/peaksql/peaksql/__init__.py: no such file or directory
Usage:
  cc-test-reporter after-build [flags]

Flags:
  -s, --batch-size int               batch size for source files (default 500)
  -e, --coverage-endpoint string     endpoint to upload coverage information to (default "https://api.codeclimate.com/v1/test_reports")
  -t, --coverage-input-type string   type of input source to use [clover, cobertura, coverage.py, excoveralls, gcov, gocov, jacoco, lcov, simplecov, xccov]
      --exit-code int                exit code of the test run
  -r, --id string                    reporter identifier (default "***")
      --insecure                     send coverage insecurely (without HTTPS)
  -p, --prefix string                the root directory where the coverage analysis was performed (default "/home/runner/work/peaksql/peaksql")

Global Flags:
  -d, --debug   run in debug mode

(node:2854) UnhandledPromiseRejectionWarning: TypeError: (s || "").replace is not a function
    at escapeData (/home/runner/work/_actions/paambaati/codeclimate-action/v2.5.2/node_modules/@actions/core/lib/command.js:66:10)
    at Command.toString (/home/runner/work/_actions/paambaati/codeclimate-action/v2.5.2/node_modules/@actions/core/lib/command.js:60:35)
    at issueCommand (/home/runner/work/_actions/paambaati/codeclimate-action/v2.5.2/node_modules/@actions/core/lib/command.js:23:30)
    at Object.issue (/home/runner/work/_actions/paambaati/codeclimate-action/v2.5.2/node_modules/@actions/core/lib/command.js:27:5)
    at Object.error (/home/runner/work/_actions/paambaati/codeclimate-action/v2.5.2/node_modules/@actions/core/lib/core.js:120:15)
    at /home/runner/work/_actions/paambaati/codeclimate-action/v2.5.2/lib/main.js:163:20
    at Generator.throw (<anonymous>)
    at rejected (/home/runner/work/_actions/paambaati/codeclimate-action/v2.5.2/lib/main.js:6:65)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
(node:2854) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:2854) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

It manages to find the coverage.xml file, but somehow something goes wrong with the paths. I am not sure what could be causing this. I am using the latest version (v2.5.2). This is how I call it:

```
    - name: Run tests
      run: |
        coverage run -m unittest discover -v test

    - uses: paambaati/codeclimate-action@v2.5.2
      env:
        CC_TEST_REPORTER_ID: ${{ secrets.CodeClimate }}
      with:
        coverageCommand: coverage xml
        debug: true
```

### PR DESCRIPTION
**Pull request by paambaati**
This PR fixes #117 and #109.

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -108,6 +108,9 @@ export function run(
           `codeclimate.${i}.json`
         ];
         if (codeClimateDebug === 'true') commands.push('--debug');
+        if (!coveragePrefix && typeof coveragePrefix === 'string') {
```
Comment (rwjblue) [inline src/main.ts]: @paambaati - As far as I can tell this logic is only going to append `--prefix` when `coveragePrefix` is `""` (empty string). I'd make a PR to address/fix but I'm not quite sure what the goal here is.  Are you trying to only add `--prefix` only when `coveragePrefix` was passed?

Comment (rwjblue) [reply src/main.ts]: Specifically, I'm seeing the following command being executed:

```js
/home/runner/work/ember-cli/ember-cli/cc-reporter format-coverage coverage/lcov.info -t lcov -o codeclimate.0.json --prefix 
```

When the configuration is:

```yml
      - name: Test && Report to Code Climate
        uses: paambaati/codeclimate-action@v2.5.4
        with:
          coverageCommand: yarn test:cover
          coverageLocations: "coverage/lcov.info:lcov"
```

Comment (paambaati) [reply src/main.ts]: > Are you trying to only add --prefix only when coveragePrefix was passed?

That is correct.

> I'd make a PR to address/fix

Much appreciated, please and thank you.

Comment (rwjblue) [reply src/main.ts]: No problem! Here is what I was thinking https://github.com/paambaati/codeclimate-action/pull/131 for the solution.

**Pull request merged**